EasyPost/Insurance.cs

```diff
@@ -98,33 +99,20 @@ public static Insurance Create(Dictionary<string, object> parameters)
                 }
             });
 
-            return request.Execute<Insurance>();
+            return await request.Execute<Insurance>();
         }
 
         /// <summary>
         ///     Retrieve an Insurance from its id.
         /// </summary>
         /// <param name="id">String representing an Insurance. Starts with "ins_".</param>
         /// <returns>EasyPost.Insurance instance.</returns>
-        public static Insurance Retrieve(string id)
+        public static async Task<Insurance> Retrieve(string id)
         {
             Request request = new Request("insurances/{id}");
             request.AddUrlSegment("id", id);
 
-            return request.Execute<Insurance>();
-        }
-
-        private static Insurance SendCreate(Dictionary<string, object> parameters)
```

Comment:
Please call the removal of this out in the CHANGELOG.

Reply:
It was internal, so no one ever saw it. Dead code.